src/GvimExt/gvimext.cpp

```diff
@@ -613,7 +622,7 @@ STDMETHODIMP CShellExt::QueryContextMenu(HMENU hMenu,
     if (cbFiles > 1)
     {
 	mii.wID = idCmd++;
-	mii.dwTypeData = _("Edit with &multiple Vims");
+	mii.dwTypeData = _("Edit with Vim using tabpages");
```

Comment:
Adding an access key might be better. E.g.:
```suggestion
	mii.dwTypeData = _("Edit with Vim using &tabpages");
```